packages/design-preview/src/components/style-variation.tsx

```diff
@@ -9,35 +8,22 @@ interface StyleVariationPreviewsProps {
 	variations: StyleVariation[];
 	selectedVariation?: StyleVariation;
 	onClick: ( variation: StyleVariation ) => void;
-	showGlobalStylesPremiumBadge: boolean;
+	description?: TranslateResult;
 	showOnlyHoverViewDefaultVariation?: boolean;
 }
 
 const StyleVariationPreviews: React.FC< StyleVariationPreviewsProps > = ( {
 	variations = [],
 	selectedVariation,
 	onClick,
-	showGlobalStylesPremiumBadge,
+	description,
 	showOnlyHoverViewDefaultVariation,
 } ) => {
 	return (
 		<GlobalStylesVariations
 			globalStylesVariations={ variations as GlobalStylesObject[] }
 			selectedGlobalStylesVariation={ selectedVariation as GlobalStylesObject }
-			premiumBadge={
-				showGlobalStylesPremiumBadge && (
-					<PremiumBadge
-						className="design-picker__premium-badge"
```

Comment:
This class is no longer used, so I think we can remove a few styles from the SCSS file

Reply:
Oh that's right. I planned to remove it but I guess I forgot about it.